express-static/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+const newPath = path.join(__dirname, 'public');
+console.log('newPath:', newPath);
+
+const servePublicFiles = express.static('public');
+
```

Comment:
We want to use the full absolute path to the public directory (your `newPath`) instead of just the string `'public'` when creating the static middleware. 

<img width="891" alt="image" src="https://user-images.githubusercontent.com/7432943/161603320-7852306c-7794-4669-9797-ca3f8403f26e.png">